### DISCUSSION TITLE
Can a pre-made configure script and its relatives be added back?

### DISCUSSION THREAD

**Discussion by tachalorah** [util-linux/util-linux#1642] [Q&A] [answered]
Hi. I have a source-based system and I'm rebuilding it since a lot of softwares have been updated. And as util-linux is one of the earliest software that I build, not having a configure script is quite inconvenience, as autoconf and meson are not built yet. My other system (Artix) also failed to build it because it had a newer gettext. However, by changing the version number it's building smooth now. 

I think the configure script shouldn't be removed, as it would prevent the version conflicts and the extra layer of difficulty that has been added to source-based users and/or (for sometimes) anyone who wanted to build the utilities and too lazy to use their brain to solve the problem they encounter. Can it be added back in a recent version soon?

**Comment by karelzak** [comment, marked as answer]
The official upstream product is tarball (from kernel.org) where is the ./configure script.
